### PR TITLE
Regen group tx

### LINKED ISSUE #616: Regen: custom tx

### PR DESCRIPTION
**Pull request by yayay927**
closes #616 

## Checklist
- [x] Ran Linting
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Wrote unit tests.
- [x] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: src/models/msg/group/msg_update_group_account_admin_request.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    this.admin = payload.admin;
+    this.newAdmin = payload.newAdmin;
+  }
+
+  static fromJson(json: any) {
+    return new MsgUpdateGroupAccountAdminRequest({
+      json,
+      type: json['@type'],
+      admin: json.admin,
+      newAdmin: json.newAdmin,
```
Comment (ryuash) [inline src/models/msg/group/msg_update_group_account_admin_request.ts:23]: `new_admin`

```
message MsgUpdateGroupAccountAdminRequest {

    // admin is the account address of the group admin.
    string admin = 1;
    
    // address is the group account address.
    string address = 2;
    
    // new_admin is the new group account admin.
    string new_admin = 3;
}
```

File: src/models/msg/group/msg_update_group_admin_request.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    this.admin = payload.admin;
+    this.newAdmin = payload.newAdmin;
+  }
+
+  static fromJson(json: any) {
+    return new MsgUpdateGroupAdminRequest({
+      json,
+      type: json['@type'],
+      admin: json.admin,
+      newAdmin: json.newAdmin,
```
Comment (ryuash) [inline src/models/msg/group/msg_update_group_admin_request.ts:23]: `new_admin`

```
// MsgUpdateGroupAdminRequest is the Msg/UpdateGroupAdmin request type.
message MsgUpdateGroupAdminRequest {

    // admin is the current account address of the group admin.
    string admin = 1;
    
    // group_id is the unique ID of the group.
    uint64 group_id = 2;
   
    // new_admin is the group new admin account address.
    string new_admin = 3;
}
```

File: public/locales/en/transactions.json [hunk 3]
```diff
@@ -15,8 +15,11 @@
   "none": "None",
   "bank": "Bank",
   "crisis": "Crisis",
+  "data": "Data",
   "distribution": "Distribution",
+  "ecocredit": "Ecocredit",
   "governance": "Governance",
+  "group": "Group",
```
Comment (ryuash) [inline public/locales/en/transactions.json:22]: Why add these if you didn't even put them in here https://docs.bigdipper.live/cosmos-based/frontend/customisations/tx#update-message-filter

**Review by ryuash — CHANGES_REQUESTED** [review]
You did not add them in `src/components/msg/utils.tsx`
https://docs.bigdipper.live/cosmos-based/frontend/customisations/tx#update-utils

File: public/locales/en/transactions.json (hunk 3 above)
Comment (yayay927) [reply public/locales/en/transactions.json:22]: I did put them in src/components/transaction_messages_filter/utils.tsx. Or you mean lack in other file?

**Review by ryuash — APPROVED** [review]

**Pull request merged**